yt/chyt/controller/internal/jupyt/controller.go

```diff
@@ -31,17 +50,27 @@ func (c *Controller) UpdateState() (changed bool, err error) {
 	return false, nil
 }
 
-func (c *Controller) buildCommand(speclet *Speclet) (command string, env map[string]string) {
+func (c *Controller) buildCommand(speclet *Speclet, oplet *strawberry.Oplet) (command string, env map[string]string) {
 	// TODO(max): take port from YT_PORT_0.
 	// TODO(max): come up with a solution how to pass secrets (token or password) without exposing them in the
 	// strawberry attributes.
+
 	cmd := fmt.Sprintf(
 		"bash -x start.sh /opt/conda/bin/jupyter lab --ip '*' --port %v --LabApp.token='' --allow-root >&2", JupytPort)
-	return cmd, map[string]string{
-		"NB_GID":  "0",
-		"NB_UID":  "0",
-		"NB_USER": "root",
+	jupyterEnv := map[string]string{
+		"NB_GID":                   "0",
+		"NB_UID":                   "0",
+		"NB_USER":                  "root",
+		"JUPYTER_YT_HOST":          c.cluster,
```

Comment:
Why `JUPYTER_YT_HOST`?
`JUPYTER_YT_PROXY` or `JUPYTER_YT_CLUSTER_PROXY` look more reasonable for me